frontend/lib/requests/user-activity.ts

```
@@ -256,19 +205,66 @@ function formatPostHogDescription(event: any): string {
     }
 
-    // Fallback
     return "Viewed page";
   }
 
-  // Handle lesson view events
+  if (eventName.toLowerCase().includes("quiz_answered_correctly")) {
+    const questionTitle = properties.question_title || "";
+    const lessonName = properties.lesson_name;
+    const dropletName = properties.droplet_name;
+    const quizType = properties.quiz_type || "";
```

ESLint

[error] 'quizType' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)